### PR TITLE
Update cheatsheet.md

### PR DESCRIPTION
**Pull request by andrzejsydor**
Fixed some typos

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@andrzejsydor
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by andrzejsydor** [comment]
I signed it

**Comment by bradtopol** [comment]
Hi @andrzejsydor Thank you for your contribution. Please sign the CLA so we can accept your PR for review.
/assign

**Comment by andrzejsydor** [comment]
rerun checks

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ kubectl rollout restart deployment/frontend                      # Rolling resta
 
 cat pod.json | kubectl replace -f -                              # Replace a pod based on the JSON passed into std
 
-# Force replace, delete and then re-create the resource. Will cause a service outage.
+# Force replaces, delete and then re-create the resource. Will cause a service outage.
```
Comment (bradtopol) [inline content/en/docs/reference/kubectl/cheatsheet.md:231]: This sentence is not grammatically correct.  Could potentially change it to 
"Force replaces, deletes, and then re-creates the resource. Will cause a service outage."

But then the verb tense doesn't match the others so not sure its worth it

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -388,7 +388,7 @@ Verbosity | Description
 --------------| -----------
 `--v=0` | Generally useful for this to *always* be visible to a cluster operator.
 `--v=1` | A reasonable default log level if you don't want verbosity.
-`--v=2` | Useful steady state information about the service and important log messages that may correlate to significant changes in the system. This is the recommended default log level for most systems.
+`--v=2` | Useful steady-state information about the service and essential log messages that may correlate to significant system changes. This is the recommended default log level for most systems.
```
Comment (bradtopol) [inline content/en/docs/reference/kubectl/cheatsheet.md:391]: I'm not sure of the purpose of this change.   What is the motivation for it?

File: content/en/docs/reference/kubectl/cheatsheet.md (hunk 1 above)
Comment (irvifa) [inline content/en/docs/reference/kubectl/cheatsheet.md:231]: This sentence is not grammatically correct. I'm wondering what's the purpose of the changes though.

**Review by irvifa — COMMENTED** [review]
I need another help from people who are native english speaker as well
/hold

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25877--kubernetes-io-master-staging.netlify.app/docs/reference/kubectl/cheatsheet/#updating-resources

File: content/en/docs/reference/kubectl/cheatsheet.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/reference/kubectl/cheatsheet.md:231]: Hi @andrzejsydor .
This original comment is confusing.
How about updating the comment to:
```
# `kubectl replace --force` deletes then recreates the resource. You can expect a service outage.
```

**Comment by tengqm** [comment]
Please feel free to reopen and resume the work.
/close
